recipes/fmt/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  5.3.0:
```

Comment:
To avoid confusions I would quote also here the versions. Not needed to change here, but remember for the future.